bindings/rust/build.rs

```diff
@@ -2,37 +2,18 @@ fn main() {
     let src_dir = std::path::Path::new("src");
 
     let mut c_config = cc::Build::new();
-    c_config.include(&src_dir);
+    c_config.include(src_dir);
```

Comment:
I'm not a Rust user but doesn't & indicate a non-owning reference? Removing it would cause c_config.include() to take ownership of `src_dir` and then further uses of `src_dir` in this function like `src_dir.join("parser.c");` would fail?

Again, not a Rust user but wanted to mention it just in case. If it's a concern I'd recommend keeping the current text

```suggestion
    c_config.include(&src_dir);
```

Reply:
src_dir is already &Path (paths are non-owning unlike PathBuf, similar to str/String)